src/main/java/de/slimecloud/slimeball/features/moderation/TimeoutListener.java

```diff
@@ -52,7 +54,7 @@ public void onTimeout(@NotNull UserTimeoutedEvent event) {
 						.addField("Endet", TimeFormat.RELATIVE.format(event.getEnd()), true)
 						.addField("Grund", event.getReason(), false)
 						.build()
-		)).queue();
+		)).queue(s -> {}, exception -> informed.set(false));
```

Comment:
1. You can pass null as first argument
2. You shouldn't catch all errors but only the one that actually causes the problem here (to still have the exception in the logs for unexpected problems). You can use the ErrorHandler class for that